spyder_profiler/widgets/profilergui.py

```diff
@@ -462,7 +462,13 @@ def load_data(self, profdatafile):
         self.profdata = stats_indi[0]
         
         if self.compare_file is not None:
-            stats_indi.append(pstats.Stats(self.compare_file))
+            try:
+                stats_indi.append(pstats.Stats(self.compare_file))
+            except IOError as e:
+                QMessageBox.critical(
+                    self, _("Error"),
+                    _("Error when trying to load profiler results"))

```

Comment:
I think there's no need to show this error message. We can simply swallow the error...

@rlaverde?

Reply:
I mean, what's the advantage for users to receive this message? Is there something they can do about it?

Reply:
Users will know that the profiler failed, if not It'll be strange that after pressing run nothing happened

Reply:
I see, but then we should also make `self.compare_file = None` before showing the message and make this method return at this point, right?

Reply:
Only making the self.compare_file = None, the rest of the method should be executed because there could be previous results that need to be loaded

Reply:
Ok, good :+1: